packages/agreement-process/test/archiveAgreement.test.ts

```diff
@@ -58,16 +63,85 @@ describe("archive agreement", () => {
     const agreementId = returnedAgreement.id;
 
     expect(agreementId).toBeDefined();
-    if (!agreementId) {
-      fail("Unhandled error: returned agreementId is undefined");
-    }
-
     const actualAgreementData = await readLastAgreementEvent(agreementId);
 
-    if (!actualAgreementData) {
-      fail("Creation fails: agreement not found in event-store");
```

Comment:
In my opinion this is redundant too, the expect below will fail if `actualAgreementData` is nullish